seed/views/main.py

```diff
@@ -2017,7 +2020,7 @@ def progress(request):
     progress_key = json.loads(request.body).get('progress_key')
 
     if cache.get(progress_key):
-        result = cache.get(progress_key)
+        result = get_cache(progress_key)
```

Comment:
Interesting. In tasks.py all the commands used on caches are get_cache and set cache. It should be cache.get and cache.set when we get all the progress stuff classified.